src/CDN/CloudFront.php

```diff
@@ -77,17 +78,50 @@ class CloudFront implements CDNInterface
     protected $client;
 
     /**
-     * @param string $path
-     * @param string $key
-     * @param string $secret
-     * @param string $distributionId
+     * @var string
      */
-    public function __construct($path, $key, $secret, $distributionId)
-    {
+    private $region;
+
+    /**
+     * @var string
+     */
+    private $version;
+
+    /**
+     * @todo: Make mandatory argument 5 and 6 when support for aws/aws-sdk-php < 3.0 is dropped.
```

Comment:
Mybe `NEXT_MAJOR` will be better? Drop support for lower version will be BC so we can drop it in any time but farther in next major.

Reply:
AFAIK, dropping support for a lower version is not considered a BC break.

Reply:
Correct, dropping support for a lower version is not considered a BC break ===  will be BC.

Anyway will be nice to mark it as `NEXT_MAJOR`, becouse in next major we should not support lower version.

Also `todo` is more like: `"HEY!!! It something to do here now. It is not ready!"`